pkg/crd/crds/backups-backuppolicy.go

```diff
@@ -34,23 +33,26 @@ const (
 )
 
 func init() {
-	if err := yaml.Unmarshal(backupsBackupPolicy, &backupsBackupPolicyCRD); err != nil {
-		panic(err)
-	}
+	mustLoadCRD(backupsBackupPolicy, backupsBackupPolicySchemaRaw, &backupsBackupPolicyCRD, &backupsBackupPolicyCRDWithSchema)
 }
 
-func BackupsBackupPolicyPolicy() *apiextensions.CustomResourceDefinition {
-	return backupsBackupPolicyCRD.DeepCopy()
+func BackupsBackupPolicyPolicy(opts ...GetCRDOptions) *apiextensions.CustomResourceDefinition {
+	return getCRD(backupsBackupPolicyCRD, backupsBackupPolicyCRDWithSchema, opts...)
```

Comment:
Just unmarshal it twice instead of creating new func

Reply:
BackupsBackupPolicyPolicy -> this should have same working function, add BackupsBackupPolicyPolicyWithOptions

Reply:
BackupsBackupPolicyPolicyWithOptions should default options to the one which is used by CRD -> for ML True, others False (if not set at all)